crates/bitwarden-vault/src/cipher/cipher.rs

```diff
@@ -844,6 +858,7 @@ mod tests {
                 reprompt: cipher.reprompt,
                 organization_use_totp: cipher.organization_use_totp,
                 edit: cipher.edit,
+                permissions: cipher.permissions,
```

Comment:
I'm considering adding more unit tests for these implementations, but I have to add `PartialEq` to a lot of models. I just wanted to check in before doing so.
1. `Encryptable<KeyIds, SymmetricKeyId, Cipher> for CipherView`
2. `Decryptable<KeyIds, SymmetricKeyId, CipherView> for Cipher`

Reply:
Not sure why you need to add PartialEq to Encryptable?

Reply:
That's correct. PartialEq is required to compare structs. I don't think we would be against adding it to CipherView though.